#Satellite Range information from Novatel OEM6 receiver

Header header

#number of observations
int32 numb_of_observ

#range information for the observed
RangeInformation[] info

